backend/notifications/serializers.py

```
@@ -0,0 +1,35 @@
+            'category_label',
+            'priority',
+            'priority_label',
+            'title',
+            'body',
+            'link_url',
+            'link_label',
+            'payload',
+            'is_broadcast',
+            'is_read',
+            'created_at',
+        ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)